eclass/distutils-r1.eclass

```diff
@@ -506,7 +506,7 @@ distutils_enable_sphinx() {
 	local deps autodoc=1 d
```

Comment:
IMO this isn't very useful churn and it might even be argued to contradict https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101. But I don't care enough to argue over it beyond noting it.

Reply:
> Use bash conditions [[ ... ]] rather than POSIX-ish [ ... ] or test builtin.
> 
> Rationale: the recommended constructs are less error-prone. Consistency avoids unnecessary changes when other developers edit the ebuild.

This change *very specifically* is and remains the bash condition. It's not a QA policy violation.

I'm kind of meh on the change too. However it was specifically noted when I added new code that naturally used the bash condition in its idiomatic single-equals form, that it was "inconsistent" with the rest of the file and therefore problematic.

Fixing all the other pre-existing instances is a direct response to that review. 🤷

Reply:
Just adhere to the existing style and don't change it for the sakeo f it.